### ISSUE TITLE
Improve support for standard MacOS keyboard shortcuts

### ISSUE THREAD

**Issue by donaldharvey** [microsoft/AzureStorageExplorer#3571]
**Is your feature request related to a problem? Please describe.**
Keyboard navigation does not work as expected on MacOS. It'd be a really big QoL improvement to be able to use `Cmd+Up/Down` to traverse a directory structure as I can in Finder. Additionally, the standard keyboard shortcut to select all text, `Cmd+A`, does not work on text inputs in the app.

**Describe the solution you'd like**
Better adherence throughout the app to standard MacOS keyboard conventions. A few specific examples:
   - `Cmd+,` should open the settings tab
   - `Cmd+Left/Right` should go back and forward instead of `Alt+Left/Right`. If focused on a text input, `Alt+Left/Right` should instead be bound to moving between words as normal in a MacOS text input
   - `Cmd+Up/Down` should open a child directory and open the parent directory respectively as in Finder or Cyberduck
   - (Nice to have) `Enter` should open the rename dialog instead of opening the directory

**Additional context**
See https://support.apple.com/en-gb/HT201236

**Comment by MRayermannMSFT** [comment]
Duplicate/merging with #2756

**Comment by hzpc-joostk** [comment]
On Mac, since a recent Azure Storage Explorer update the <kbd>Cmd ⌘</kbd> key is bound to opening a context menu. This is also triggered when switching windows with <kbd>Cmd ⌘</kbd> + <kbd>tab</kbd> or <kbd>Cmd ⌘</kbd> + <kbd>`</kbd>.

An improvement would be to disable this. 😉
